docs/reference-guide/graph-algorithms.md

```diff
@@ -312,4 +313,79 @@ property equal to `false` and to nodes with a `drinks_USD` property less than `1
 ```cypher
 MATCH path=(n {id: 0})-[*WSHORTEST (r, n | n.total_USD) total_weight (r, n | r.eu_border = false AND n.drinks_USD < 15)]-(m {id: 46})
 RETURN path,total_weight;
+```
+
+## All Shortest Paths
+
+Finding all shortest paths is an expansion of the weighted shortest paths problem. The goal
+of finding the shortest path is obtaining any minimum sum of weights on the path from one
+node to the other. However, there could be multiple similar-weighted paths, and this algorithm
+enables to fetch them all.
+
+Weighted shortest path implementation returns only one-row resulting path from one
```

Comment:
Do not really understand what does this "resulting" means?

Reply:
one shortest path is returned as a result, as compared to all shortest paths